tests/base_tests.py

```diff
@@ -69,6 +71,39 @@ def get_resp(
     return resp.data.decode("utf-8")
 
 
+def post_assert_metric(
```

Comment:
Is this the same but on a different place?

Reply:
yes same code, moved it into this function for reuse in the pytests